### PR TITLE
update example to use keycloak and backend demo

### PR DESCRIPTION
**Pull request by edewit**
as discussed on irc the controller demo is old and we could also demonstrate with keycloak and our backend cookbook.

### CODE REVIEW

**Comment by edewit** [comment]
@abstractj updated

**Comment by matzew** [comment]
@abstractj @edewit can we merge this?

**Comment by edewit** [comment]
:+1:

**Comment by abstractj** [comment]
@matzew @edewit the project was moved to https://github.com/aerogear/aerogear-android-cookbook/tree/master/Two-Factor. I think @danielpassos can confirm.

**Comment by danielpassos** [comment]
The old [aerogear-otp-android-demo](https://github.com/aerogear/aerogear-otp-android-demo) was moved to the [cookbook](https://github.com/aerogear/aerogear-android-cookbook/tree/master/Two-Factor)

**Comment by edewit** [comment]
right @abstractj @danielpassos, but in this pr I've change the instructions to no longer use http://controller-aerogear.rhcloud.com/aerogear-controller-demo/login but keycloak with our shoot 'n share backend to test the otp. Or should this be completely removed?

File: README.md [hunk 1]
```diff
@@ -43,14 +43,13 @@ This API is compatible with Google Authenticator apps available for [Android](ht
...
-4. Go to *Add account*
-5. Then ask to *Scan barcode*
-6. After that *john* account might be added 
-7. On your web browser go to *Try Google authenticator*
-8. Enter *username: john* and *password: 123* and use the current OTP on your mobile
+1. Follow directions to install [Shoot 'n Share backend](https://github.com/aerogear/aerogear-backend-cookbook/blob/master/Shoot/README.md)
+1. Open the [console shoot-realm credentials](http://localhost:8080/auth/admin/master/console/#/realms/shoot-realm/required-credentials) and add totp in `Required User Credentials`
+1. Open Shoot 'n Share backend http://localhost:8080/shoot/photos
+1. Login with username: *user* and password: *password*.
+1. Now open [android otp client application](https://github.com/aerogear/aerogear-otp-android-demo) on your phone
```
Comment (abstractj) [inline README.md]: Just change the URL here to point to new repository https://github.com/aerogear/aerogear-android-cookbook/tree/master/Two-Factor, please.

Also, replace all the references to *otp* by OTP.

File: README.md [hunk 2]
```diff
@@ -43,14 +43,13 @@ This API is compatible with Google Authenticator apps available for [Android](ht
...
-1. Go to [http://controller-aerogear.rhcloud.com/aerogear-controller-demo/login](http://controller-aerogear.rhcloud.com/aerogear-controller-demo/login)
-2. Login with *username: john* and *password: 123*. Click on *Give it a try*.
-3. Now open google authenticator application at your phone
-4. Go to *Add account*
-5. Then ask to *Scan barcode*
-6. After that *john* account might be added 
-7. On your web browser go to *Try Google authenticator*
-8. Enter *username: john* and *password: 123* and use the current OTP on your mobile
+1. Follow directions to install [Shoot 'n Share backend](https://github.com/aerogear/aerogear-backend-cookbook/blob/master/Shoot/README.md)
+1. Open the [console shoot-realm credentials](http://localhost:8080/auth/admin/master/console/#/realms/shoot-realm/required-credentials) and add totp in `Required User Credentials`
```
Comment (abstractj) [inline README.md]: Replace *totp* by TOTP, please.

File: README.md [hunk 3]
```diff
@@ -43,14 +43,13 @@ This API is compatible with Google Authenticator apps available for [Android](ht
...
+1. Open Shoot 'n Share backend http://localhost:8080/shoot/photos
+1. Login with username: *user* and password: *password*.
+1. Now open [android otp client application](https://github.com/aerogear/aerogear-otp-android-demo) on your phone
+1. Then scan *Scan barcode*
```
Comment (abstractj) [inline README.md]: There's no barcode, but only QRCode. Please, replace to avoid confusion.

**Comment by abstractj** [comment]
@matzew @danielpassos @edewit IMO this change does not make sense here, but only at our demos. People can't run aerogear-otp-java and scan the QRCode, but they can do it at our cookbooks.

Don't want to sound picky, but maybe we should move to the cookbooks? Anyways I can do it by myself and keep @edewit ownership if we agree on it.

**Comment by matzew** [comment]
yeah, I think makes more sense to document these instructions at the cookbook level

**Comment by edewit** [comment]
+1

**Comment by matzew** [comment]
closing PR based on discussion

**Comment by abstractj** [comment]
@edewit @matzew @danielpassos landed here https://github.com/aerogear/aerogear-android-cookbook/commit/e06473c53e85360b2432bf647ebb78626ef40ee4 with some little adjusts https://github.com/aerogear/aerogear-android-cookbook/commit/ca2f1bb20149c95509878f46d2716c1edc88ac43

**Pull request closed without merging**